### ISSUE TITLE
Show team animal badge in royal

### ISSUE THREAD

**Issue by neyl66** [codecat/tm-better-chat#62]
Hello, i think it would be useful if in royal it would show team's animal badge in front of username.

What do you think?

**Comment by codecat** [comment]
Good idea! I'm not sure if we can show the team icons per se, but we might be able to show the team name instead, given that the information is actually available. Matchmaking seems to be down for maintenance right now so I can't test it 😜
